packages/sdk-multichain/src/multichain/rpc/client.test.ts

```diff
@@ -231,24 +298,6 @@ t.describe('RPCClient', () => {
 				});
 			});
 
-			t.it('should redirect to provider for methods in METHODS_TO_REDIRECT', async () => {
-				const redirectOptions: InvokeMethodOptions = {
-					scope: 'eip155:1' as Scope,
-					request: {
-						method: RPC_METHODS.ETH_SENDTRANSACTION,
-						params: { to: '0x123', value: '0x100' },
-					},
-				};
-				mockTransport.request.mockResolvedValue({ result: '0xhash' });
-				const result = await rpcClient.invokeMethod(redirectOptions);
-				t.expect(result).toBe('0xhash');
-				t.expect(mockTransport.request).toHaveBeenCalledWith({
-					method: 'wallet_invokeMethod',
-					params: redirectOptions,
-				});
-				t.expect(mockFetch).not.toHaveBeenCalled();
```

Comment:
should we preserve this test but make it
`should redirect to provider for EVM methods not in EVM_RPC_PASSTHROUGH_METHODS` ?

Reply:
there's one that tests this via `personal_sign`. Perhaps we just rename that test title?

Reply:
yeah lets rename it